src/Registration/Registration.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import firebase from "firebase";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./Registration.css";
+import { Alert, Button, Card, Container, Form } from "react-bootstrap";
+
+export default function Registration() {
+  const [user, setUser] = useState({
+    displayName: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+    error: "",
+  });
+  const [createdState, setCreatedState] = useState(false);
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    // TODO bootstrap validation instead of validation below?
```

Comment:
Could you elaborate on this?

Reply:
Saw this https://react-bootstrap.github.io/components/forms/#forms-validation and thought it might be better, but I couldn't really find anything on password confirmation. It was a suggestion.